Gopkg.lock

```diff
@@ -79,16 +79,17 @@
 
 [[projects]]
   branch = "master"
-  digest = "1:06a0d9b156b459a988b70c9f5d7ce3dc0a0231fe5329eb56dd1cfd7704c6fa19"
+  digest = "1:604d895535c9288c1566e6146272454adccfb1b116a5fc5e3758f9d547aa4dcb"
   name = "github.com/manifestival/manifestival"
   packages = [
     ".",
+    "fake",
```

Comment:
Can you re-run `dep ensure`? This doesn't seem to be needed.

Reply:
Done.